apps/web-mzima-client/src/app/shared/components/toolbar/toolbar.component.html

```diff
@@ -105,6 +105,7 @@
           </span>
         </a>
       </ng-container>
+      <app-account-and-logout [isLink]="true"></app-account-and-logout>
```

Comment:
Very nice @MugoBrian You got the injection idea. Watch out for a PR I will be sending (to your branch) for you to take a look at and that you would need to merge.

Reply:
Still waiting.
@Ifycode

Reply:
Change of plans. Will be merging and resolving in a branch that I will create instead.